<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonorRegister</class>
 <widget class="QDialog" name="DonorRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DonorRegister{
background-color: rgb(224, 241, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>501</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{

	
    background: #fff;
    margin: 0 auto;
    border-radius: 20px;
	background-color:rgb(70,130,180);

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
    display: block;
    font-size: 1.5em;
    font-weight: bold;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitDonor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#submitDonor{
background-color: white;

    border-width: 2px;
    border-radius: 10px;

    font: bold 14px;

    padding: 6px;
}

#submitDonor:hover{
background-color: rgb(224, 241, 255);
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateLastDonation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#dateLastDonation{


border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordDonor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#passwordDonor{

border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DonorSuffer">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>None</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blood Pressure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Disorders</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thyroid Disease</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diabetes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cancer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heart Disorders</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hepatitis</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="NameDonor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#NameDonor{

border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;}
</string>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Do you suffer from ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="AgeLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GenderDonor">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>73</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Blood Type</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DonorAge">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="bloodTypeDonor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="emailDonor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#emailDonor{

border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;

}</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Do you take any medicine</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Date of last donation</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="medicinDonor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Write here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
